css/styles.css

```diff
@@ -17,3 +17,18 @@
   width: 19%; }
 
 /*# sourceMappingURL=styles.css.map */
```

Comment:
Looks like the added CSS comes after the SASS mapping. You'll want to make sure to add any new css to the scss file and use grunt to push it to the css automatically or it'll get overwritten.